src/Components/NFTsCardBrowse/index.css

```diff
@@ -37,14 +37,15 @@
 }
 
 .content-list-nft .ant-card .ant-card-cover {
-  padding: 1rem;
+  padding: 0;
 }
 
 .content-list-nft .ant-card .ant-card-cover img {
   height: 20vh;
   margin: 0 auto;
-  border-radius: 5px;
-  object-fit: contain;
+  border-radius: 5px 5px 0 0;
+  object-fit: cover;
```

Comment:
Image will be broken if using this attribute imo

Reply:
|It depends on what do you mean by broken. The gallery view could use image preview in zoomed mode. Its up to you but I prefer this simple elegant way of having a sneak peek into what is an image.
If user clicks it then he will get the full picture.
Actually I can propose one more way of displaying the full image and using it additionally as blurred card background to fill the whitespaces.